### PR TITLE
Avoid checking the version on a Language Server path from manual configuration

### PR DESCRIPTION
**Pull request by ricardo-espinoza**

### CODE REVIEW

**Review by anjbur — CHANGES_REQUESTED** [review]
I ran the SDL pipeline against this change, but it's still resulting in the same bug detected. Is that bug no longer valid after these changes?

File: src/VSCodeExtension/src/languageServer.ts [hunk 1]
```diff
@@ -134,42 +136,54 @@ export class LanguageServer {
...
+                throw new Error(`Language server returned error when reporting version: ${response.stderr}`);
+            }
+
+            version = response.stdout.trim();
+            let info = getPackageInfo(this.context);
+            if (info === undefined || info === null) {
+                throw new Error("Package info was undefined.");
+            }
+            console.log(`[qsharp-lsp] Package is ${info.name} with ${info.nugetVersion} and ${info.version}`);
+            if (versionCheck && info.nugetVersion !== version) {
```
[the remark this answers is not in the document]
Comment (ricardo-espinoza) [reply src/VSCodeExtension/src/languageServer.ts:178]: Oops, that remained from a previous version. Let me fix, thanks for the catch!

**Comment by ricardo-espinoza** [comment]
> I ran the SDL pipeline against this change, but it's still resulting in the same bug detected. Is that bug no longer valid after these changes?

Uhm, I was hoping that it would go away because of the scoping down of the path, but I'll try something else now.

**Review by cgranade — APPROVED** [review]

**Review by anjbur — APPROVED** [review]
The SDL pipeline is running clean and this looks great! Thank you for making these changes

**Pull request merged**